routingpy/routers/graphhopper.py

```diff
@@ -429,7 +429,7 @@ def isochrones(
         intervals,
         type="json",
         buckets=1,
-        interval_type=None,
```

Comment:
we'll default explicitly to `time` for all providers now, otherwise it could be `None`